spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/cf-tasks.adoc

```diff
@@ -2,112 +2,125 @@
 = Tasks on Cloud Foundry
 
 Spring Cloud Data Flow's task functionality exposes new task capabilities within
-the Pivotal Cloud Foundry runtime. It is important to note that the current underlying PCF
+the Pivotal Cloud Foundry runtime.
+
+IMPORTANT: The current underlying PCF
 task capabilities are considered experimental for PCF version versions less than 1.9.  See
- <<enable-disable-specific-features>> for how to disable task support in Data Flow.
+<<enable-disable-specific-features>> for how to disable task support in Data Flow.
 
 == Version Compatibility
 
 The task functionality depends on the latest versions of PCF for runtime support. This
-release requires PCF version 1.7.12 or higher to run tasks.  Tasks are an experimental
+release requires PCF version 1.7.12 or higher to run tasks. Tasks are an experimental
 feature in PCF 1.7 and 1.8 and a GA feature in PCF 1.9.
 
 == Task Database Schema
 
-The database schema for Task applications was changed slighlty from the 1.0.x to 1.1.x version of
-Spring Cloud Task.  Since Spring Cloud Data Flow automatically creates the database schema if it is
-not present upon server startup, you may need to update the schema if you ran a 1.0.x version of the
-Data Flow server and now are upgrading to the 1.1.x version.  You can find the migration scripts
+The database schema for Task applications was changed slightly from the 1.0.x to 1.1.x version of
```

Comment:
The current version is at v1.6. This note is very old as well. We don't want to mention this anymore. The older release ref. guides will include it anyway. If someone asks for it, we can link to the old docs and advise them to upgrade ASAP.